server/routerlicious/packages/services-client/src/restWrapper.ts

```diff
@@ -83,6 +83,14 @@ export class RestWrapper {
                     debug(`request to ${options.url} failed ${error ? error.message : ""}`);
                 }
 
+                if (error.response && error.response.status
+                    && error.response.data && error.response.data.retryAfter
+                    && error.response.status === 429 && error.response.data.retryAfter > 0) {
+                    setTimeout(async () => {
+                        return this.request<T>(options, statusCode);
+                    }, error.response.data.retryAfter * 1000);
+                }
```

Comment:
If we're intending to quietly retry, the function should short-circuit after setting up the timeout so that an error response is not sent